client/build.gradle

```diff
@@ -17,7 +17,7 @@ configurations {
 }
 
 dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter:3.1.5'
```

Comment:
Why?

Reply:
Cause I don't use spring boot starter web in Client

Reply:
You just deleted version tag, don't you?

Reply:
No, I changed spring-boot-starter-web dependency to spring-boot-starter